includes/components/class-review.php

```diff
@@ -160,6 +161,25 @@ public function update_rating( $review_id, $review ) {
 			return;
 		}
 
+		// Get user.
```

Comment:
please move this to a new branch created for the "notify users when their review is approved" feature, this way you can re-use the same review status change callback. then delete this branch and add comment to the Asana task

Reply:
the hook which is used in "notify users when their review is approved" feature branch is specific only for manual review approve (and others actions with comment in admin panel). it does not work when moderation is not needed